scalafmt-tests/src/test/resources/scala3/OptionalBraces_keep.stat

```diff
@@ -1407,3 +1407,22 @@ val request: Request = Request(
   new:
      method = HttpMethod.POST
 )
+<<< lambda
+rewrite.scala3.removeOptionalBraces = yes
+===
+val f: Int => Int = (x: Int) => {
+  x * x
+}
+>>>
+val f: Int => Int = (x: Int) => x * x
```

Comment:
maybe, in a separate PR, we could fix this. there was a line break in the source, but here it disappeared (and this is `keep`).